boards/arm/stm32h747i_disco/stm32h747i_disco_m4_defconfig

```diff
@@ -5,7 +5,7 @@ CONFIG_BOARD_STM32H747I_DISCO_M4=y
 CONFIG_SOC_SERIES_STM32H7X=y
 CONFIG_SOC_STM32H747XX=y
 CONFIG_CORTEX_M_SYSTICK=y
-CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=100
```

Comment:
IMHO, "default" means, you wouldn't have to set it here :)

Reply:
Doh! Git amend issue (Besides 100 is wrong), update on its way.

Reply:
Thanks @ioannisg !